### PR TITLE
add a time field to the log entries

### PR DESCRIPTION
**Pull request by ekristen**
Title says it all, resolves https://github.com/hashicorp/vault/issues/493

### CODE REVIEW

**Comment by sethvargo** [comment]
Hi @ekristen 

It looks like the tests need to be updated as well.

**Comment by ekristen** [comment]
@sethvargo there we go.

**Comment by armon** [comment]
Looks great! Thanks!

**Comment by ekristen** [comment]
+1 thanks!

**Comment by mfischer-zd** [comment]
Why UTC?  IMO localtime should be used.  If the user wants UTC he/she can set the TZ environment variable before launching vault, or set the system timezone accordingly.

**Comment by ekristen** [comment]
I disagree, here is why ...

TL;DR - consistency ... it is the universally agreed upon time standard. UTC should always be used for servers and logs (especially security). When reviewing logs, your parser should convert to local time if needed.

Servers and application logs should always be UTC. There are too many inconsistencies with using timezones as the default for an application or a server. Some timezones have daylight savings, others do not, which has been known to cause many a problem when the time changes. 

UTC is universal, it is the world wide agreed upon time standard, no matter how many servers or applications that are generating logs, 1700 hours in the app1 vs app100 vs server1 vs server100 is the same.

**Comment by mfischer-zd** [comment]
I agree with you.  However, it is not our decision to force upon the user.
